css/main.css

```diff
@@ -1,3 +1,248 @@
+  font-size: 2rem;
+}
+.main-centre {
+  margin-top: 30px;
+  font-size: 1em;
+}
+.main-centre p {
+  text-align: center;
+}
+.about-section {
+  margin: 15px;
+}
+.row2 {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: 0px;
+  margin-left: 0px;
+}
+.about-col-6 {
+  position: relative;
+  min-height: 1px;
+  padding-right: 0px;
+  padding-left: 0px;
+}
+.about-col-6 h2 {
+  font-size: 80px;
+}
+.aboutus-text {
+  margin: 5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+body {
+  font-size: 20px;
+}
+.content {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.aboutus-text p {
+  color: white;
+}
+.about {
+  background-color: rgba(0, 0, 0, 0.3);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.3em;
+  text-align: center;
+  font-size: 1.3em;
+  text-align: center;
+}
+.image-about {
+  padding: 0;
+}
+.about h1 {
+  text-align: center;
+  color: red;
+}
 
-/* Dont' forget to link this file to your HTML in the <head> */
+h5 {
+  text-align: center;
+  color: rgb(26, 25, 24);
+}
+h2 {
+  text-align: center;
+  color: rgb(138, 25, 25);
+}
+h1 {
+  color: white;
+  font-size: 3rem;
+}
+img {
+  width: 100%;
+}
+h3 {
+  color: rgb(138, 25, 25);
+  font-size: 2rem;
+}
+h4 {
+  text-align: center;
+  font-weight: bold;
+}
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -15px;
+  margin-left: -15px;
+}
+li a {
+  // display: inline-block;
+  //text-align: center;
+  text-decoration: none;
+  color: #5a413f;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+}
+.top {
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+  align-items: center;
+}
+.logo {
+  display: flex;
+  justify-content: center;
+}
+/*
+.top-navigation {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+*/
+.shop-now {
+  width: 9rem;
+  background-color: #f15a29;
+  border-color: #f15a29;
+  text-shadow: none;
+  border-style: none;
+  border-radius: 4px;
+  padding: 1rem 1.5rem;
+  color: white;
+  font-size: 1.125rem;
+}
+.send-button {
+  background-color: #f15a29;
+  border-color: #f15a29;
+  text-shadow: none;
+  border-style: none;
+  color: white;
+  font-size: 1.125rem;
+  text-align: center;
+}
+
+.main {
+  height: 50vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: url('/images/NewBirthdayCake.jpg') center no-repeat;
+}
+.last-name {
+  text-align: left;
+  color: black;
+}
+
+.submit {
+  margin: 0 auto;
+  padding: 5px 15px;
+}
+
+.section-footer {
+  margin-top: 80px;
+  background-color: rgba(0, 0, 0, 0.1);
+}
+div {
+  display: block;
+}
+h3 {
+  display: block;
+  font-size: 1.17em;
+}
+
+p {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
```

Comment:
This is default browser styling, you don't need this.
What you can do however, is override these default styles with your own, for example by setting margin to `0` for all `p` tags`.